projects/batfish-common-protocol/src/main/java/org/batfish/datamodel/IpAccessListLine.java

```diff
@@ -12,6 +14,10 @@
 
   private String _name;
 
+  private String _inInterfaceName;
```

Comment:
This should not go in IpAccessListLine. This heavily clashes with existing semantics. To filter by input interface, you simply apply the appropriate acl to the appropriate interface in the incoming direction. I understand you are trying to add support for iptables, but this is too hackish. IpAccessListLine is only supposed to filter by header bits (and classifiers).